lib/generators/cache/database/templates/create_table_for_cache.rb

```diff
@@ -7,15 +7,8 @@ def change
       t.timestamp :expires_at
     end
 
-    if mysql?
-      # MySQL and MariaDB don't support partial indexes
-      add_index :activesupport_cache_entries, :expires_at
-      add_index :activesupport_cache_entries, :version
-    else
-      # For Sqlite3, PostgreSQL we use partial index, because expires_at column can be null it would be wasteful to include it in index
-      add_index :activesupport_cache_entries, :expires_at, where: 'expires_at IS NOT NULL'
-      add_index :activesupport_cache_entries, :version, where: 'version IS NOT NULL'
-    end
+    add_index :activesupport_cache_entries, :expires_at, where: 'expires_at IS NOT NULL'
```

Comment:
@dim does it mean, that MySQL supports partial indexes? (Sorry for naive question, I haven't touched MySQL for a while).

Reply:
It doesn't, but `:where` clauses are ignored where they are not supported anyway https://github.com/rails/rails/blob/55412cd9257dc27a8a9175529857ce5f2d81f92f/activerecord/lib/active_record/connection_adapters/abstract/schema_creation.rb#L115

Reply:
That's great!

Should we leave a note, saying that some databases don't support partial indexes? Just to make it more obvious to consumers of this gem.